tests/test_queries/test-query-10.txt

```diff
@@ -0,0 +1,12 @@
+SELECT job_id,AVG(salary)
```

Comment:
Unless you want them formatted differently for a specific reason (e.g. to test different formulations), can you have the test queries conform to our SQL formatting guidelines?

Reply:
Oh right, I was actually trying to test different formulations / queries that don't fit the formatting guidelines. Do you think thats not necessary?

Reply:
Nope, that makes sense to me. Carry on!